templates/registration/logout.html

```diff
@@ -0,0 +1,6 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <p>Logged Out</p>
+  <a href="{% url "logout" %}">Click here to log in again.</a>

```

Comment:
The URL `{% url "logout" %}` is incorrect for a login link. It should be `{% url "login" %}` to direct users to the login page after they have logged out.